ci-tools/github-runner/cmd/rtool/main.go

```diff
@@ -45,6 +45,24 @@ func main() {
 		if err != nil {
 			log.Fatal(err)
 		}
+	} else if os.Args[1] == "download_artifact" {
```

Comment:
I think you'll want to update the `usage()` function, and ideally, with this many arguments, I'd prefer to use something like `flag`.

Reply:
Updated usage().